File: RelacjeSportowe.DataAccess/Models/TransmissionEvent.cs
using RelacjeSportowe.DataAccess.Interfaces;

namespace RelacjeSportowe.DataAccess.Models
{
    public class TransmissionEvent : IEntity
    {
        public int Id { get; set; }

        public string Description { get; set; }

        public string TimeInfo { get; set; }

        public TransmissionEventType TransmissionEventType { get; set; }

        public int TransmissionEventTypeId { get; set; }

        public virtual Transmission Transmission { get; set; }

        public int TransmissionId { get; set; }
    }
}

File: RelacjeSportowe.Web/ClientApp/src/app/models/transmission-event.ts
export class TransmissionEvent {
  public id: number;
  public description: string;
  public timeInfo: string;
  public transmissionEventType: string;
  public transmissionEventTypeId: number;

  constructor(data: any) {
    this.id = data.id;
    this.description = data.description;
    this.timeInfo = data.timeInfo;
    this.transmissionEventType = data.transmissionEventType;
    this.transmissionEventTypeId = data.transmissionEventTypeId;
  }
}